docs/installation/debian.md

```diff
@@ -63,12 +63,20 @@ from the new repository:
 
      The possible entries are:
 
-         # Debian Wheezy
-        deb https://apt.dockerproject.org/repo debian-wheezy main
-        # Debian Jessie
-        deb https://apt.dockerproject.org/repo debian-jessie main
-        # Debian Stretch/Sid
-        deb https://apt.dockerproject.org/repo debian-stretch main
+     - On Debian Wheezy
+
+            # Debian Wheezy
```

Comment:
nit: perhaps we should remove the comments inside the code block, because they're only repeating the line before